src/Extensions/Abstractions/Accessors/Accessor.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.Collections.Generic;
 
-namespace Microsoft.Omex.Extensions.Hosting.Services
+namespace Microsoft.Omex.Extensions.Abstractions.Accessors
```

Comment:
Maybe Extensions.DependencyInjection.Accessors? How many accessors do we have? Would simply Extensions.DependencyInjection work?

Reply:
I've put only classes required for Accessor initialization (ex. `Accessor` or `IAccessorSetter`) into this namespace to separate it from `IAccessor` class that needed to consume Accessor.

Reply:
We could put it into `Microsoft.Omex.Extensions.Abstractions.DependencyInjection ` although I'm not sure what else we would put there

Reply:
I think it's better to keep it separate